### DISCUSSION TITLE
Organizing Keystone project with larger scale project

### DISCUSSION THREAD

**Discussion by tyteen4a03** [keystonejs/keystone#7481] [Questions] [answered]
Something that I don't really see in the Keystone docs is how you are supposed to organize your `keystone.ts` when your project gets big. 

When you get to a certain size (like 10 types, each with its own business logics) putting everything within `keystone.ts` quickly falls apart. What's the official way of splitting up the various parts to make them more manageable as the project grows?

**Comment by raveling** [comment, marked as answer]
@tyteen4a03 there's no strongly recommended way to do this.

You could have a `lists` directory with a file for each list. Our [eCommerce example](https://github.com/keystonejs/keystone/tree/main/examples-staging/ecommerce/schemas) splits the lists into individual files inside a `schemas` directory.
